src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -136,7 +256,9 @@ private void tickChunk(@Nonnull Set<BlockTicker> tickers, @Nonnull String chunk)
         }
     }
 
-    private void tickLocation(@Nonnull Set<BlockTicker> tickers, @Nonnull Location l) {
+    private void tickLocation(@Nonnull List<BlockTicker> tickers, @Nonnull Location l, @Nonnull BlockingQueue<Runnable> syncTasks) {
+
+
```

Comment:
What's with the two empty lines here...?

Reply:
Traces from removes debug messages